### ISSUE TITLE
New Updates of ATAK

### ISSUE THREAD

**Issue by tomcha34** [deptofdefense/AndroidTacticalAssaultKit-CIV#308]
Are we able to get an update from ATAK V4.6? Ideally we would catch up to the new release of V4.10. Thank you

**Comment by talentedbrute** [comment]
Yes this would be helpful to understand the new plugin structure.  When can we expect access to gitlab on tak.gov with the release of 5.0?

**Comment by bisgroves** [comment]
The old plugin structure is still supported via a more concise mechanism and will still ship with ATAK during the 5.X period if not longer.

**Comment by talentedbrute** [comment]
@bisgroves Are you saying that the pre-4.9 plugin architecture will work in 4.9 and higher?  We don't need to convert to the new AbstractPlugin class?

**Comment by bisgroves** [comment]
Correct, the changes you might need will be minor.

**Comment by talentedbrute** [comment]
@bisgroves So updates to the new AbstractPlugin are still required for 4.9 and later, got it.  Any idea when 5.0 source will be available to CIV developers via GitHub or git.tak.gov per the 2023 TAK Offsite brief given by Ryan?

**Comment by niccellular** [comment]
Public access to modern ATAK source would be nice

**Comment by bisgroves** [comment]
We currently have hit technical roadblocks our interactions with git hub that make this non-trivial.   That being said, I believe that there are efforts in progress to move forward.   More to follow.

**Comment by FarrantAlex** [comment]
Sounds like middle management.
The old men blocking this will say it’s for the greater good but they just don’t understand it so have defaulted to inaction. 
Luckily our enemies also have stubborn old men in charge :)

**Comment by bisgroves** [comment]
Sadly its not that easy but really comes down to a technical issue with how we use github.

**Comment by talentedbrute** [comment]
I thought, based on Ryan's presentation at the Off-Site in 2023, TPC was moving away from pushing changes to GitHub and opening up the git.tak.gov for CIV (read-only) to reduce the overhead of syncing the two repos.

**Comment by bisgroves** [comment]
That is all true - the technical issue increased the syncing burden.

**Comment by tomcha34** [comment]
This has been frozen for a long time. I know TPC has made a commitment to open source, and I am not seeing that. Please provide a meaningful update.
